### PR TITLE
task three from IO is ready

### PR DESCRIPTION
**Pull request by ivanP91**
task three is ready for review

### CODE REVIEW

File: src/com/clouway/File.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+public class File {
+  private String file;
+  private String breakCharacter;
+
+  public File(String file, String breakCharacter) {
+    this.breakCharacter=breakCharacter;
+    this.file=file;
+  }
+
+  public void write() throws IOException {
```
Comment (mgenov) [inline src/com/clouway/File.java]: write what?

Comment (ivanP91) [inline src/com/clouway/File.java]: writing  input data 
i change name method on introduce

Comment (mgenov) [inline src/com/clouway/File.java]: ?

Comment (ivanP91) [inline src/com/clouway/File.java]: write(String file,String breakCharacter)

Comment (mgenov) [inline src/com/clouway/File.java]: File will write to another fill a breaking character

Comment (ivanP91) [inline src/com/clouway/File.java]: writeUntilBreakCharacter(String file,String breakCharacter)

Comment (mgenov) [inline src/com/clouway/File.java]: write what ?

Comment (ivanP91) [inline src/com/clouway/File.java]: writeText

**Pull request closed without merging**